transport.go

```diff
@@ -0,0 +1,103 @@
+package rollrus
```

Comment:
If this file were put in ./internal/transport/transport.go the extra bits of "transport" could be removed from the various names.

Reply:
Good call, done in 1842e48.